runtime/queries/supercollider/highlights.scm

```diff
@@ -0,0 +1,101 @@
+; highlights.scm
+; See this for full list: https://github.com/nvim-treesitter/nvim-treesitter/blob/master/CONTRIBUTING.md
```

Comment:
Helix uses different capture names than neovim. See https://docs.helix-editor.com/master/themes.html#syntax-highlighting

For example captures like `parameter` need to become `variable.parameter`, `property` becomes `variable.other.member`, etc.